doc/icache.rst

```diff
@@ -240,6 +240,9 @@ On other cycles, the address counter will be incremented every time an instructi
 This increment depends on the instruction data (visible at ``rdata_o``): if the bottom bits of the instruction are not ``2'b11``, the instruction is considered to be compressed and the address will be incremented by 2.
 If the bottom bits of the instruction are ``2'b11`` then the instruction is considered to be uncompressed and the address will be incremented by 4.
 Since the contents of ``rdata_o`` are not specified if an instruction fetch has caused an error, the core must signal a branch before accepting another instruction after it sees ``err_o``.
+The address in error is that signalled on ``addr_o``.
```

Comment:
I found this quite confusing to read. Would it make more sense just to say that a transaction on the `icache <-> core` bus pertains to data starting at `addr_o`? Then I feel like explaining the difference between `err_o` and `err_plus2_o` is kind of separate. (Maybe even a separate paragraph: this is my fault, but I think that the text is pretty dense!)

Reply:
I've had another go, creating a separate paragraph and removing the first sentence, since the meaning of ``addr_o`` is already specified elsewhere.